lib/xud/XudInterface.ts

```diff
@@ -0,0 +1,8 @@
+import BaseClient from '../BaseClient';
+
+/**
+ * A generic interface that can be used for multiple Lightning clients and implementations
+ */
+interface XudInterface extends BaseClient {}
```

Comment:
I don't think we will need this interface since there is just one implementation of the XUD protocol